### PR TITLE
Ref Guide - Cobbler

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10587

### CODE REVIEW

File: modules/reference/pages/admin/cobbler.adoc [hunk 1]
```diff
@@ -1,8 +1,7 @@
 [[s3-sattools-config-cobbler]]
 = Cobbler
 
-On the menu:Admin[Manager Configuration > Cobbler] page you can run the Cobbler synchronization by clicking btn:[Update].
+The menu:Admin[Manager Configuration > Cobbler] page allows you to run a Cobbler synchronization.
+You can repair or rebuild the contents of [path]``/srv/tftpboot`` or [path]``/srv/www/cobbler`` after a manual modification of the Cobbler setup.
```
Comment (keichwa) [inline modules/reference/pages/admin/cobbler.adoc]: sth completely different... this reminds me.  We should either add a trailing slash to a directory path (/srv/tftpboot/) or put the literal word "directory" there
```
the [path]``/srv/tftpboot`` directory
```
(to be adjusted to the actual phrase, of course)

File: modules/reference/pages/admin/cobbler.adoc [hunk 2]
```diff
@@ -1,8 +1,7 @@
...
 
-image::admin_configuration_cobbler.png[scaledwidth=80%]
-
-Cobbler synchronization is used to repair or rebuild the contents of [path]``/srv/tftpboot`` or [path]``/srv/www/cobbler`` when a manual modification of the cobbler setup has occurred.
+For more information on Cobbler, see xref:client-configuration:cobbler.adoc[].
```
Comment (keichwa) [inline modules/reference/pages/admin/cobbler.adoc]: Wondering whether `information on` and `information about` are equally fine?  `on` is shorter ;)

Comment (Loquacity) [reply modules/reference/pages/admin/cobbler.adoc]: `about` is probably better, honestly. I think it meets STE guidelines.

File: modules/reference/pages/admin/cobbler.adoc (hunk 1 above)
Comment (Loquacity) [reply modules/reference/pages/admin/cobbler.adoc]: or both?

**Comment by keichwa** [comment]
Lana Brindley <notifications@github.com> writes:

> or both?

Maybe, we should start with asking for both (then we'll end up
with at least one and hopefully not with a bad mixture of both):

    Find it in the [path]``/tmp/``.

;)

-- 
Karl Eichwalder                    SUSE Software Solutions Germany GmbH
Product & Solution Documentation   Maxfeldstraße 5
                                   90409 Nürnberg, Germany
Geschäftsführer: Felix Imendörffer (HRB 36809, AG Nürnberg)

**Pull request merged**